Documentation/guides/BuildProcess.md

```diff
@@ -1135,6 +1135,29 @@ server, the following MSBuild properties can be used:
 
     Note: `env:` is not supported when using `AndroidPackageFormat`=`aab`. 
 
+-   **JarsignerTimestampAuthorityUrl** &ndash; This property
+    allows you to specify a Url to a timestamp authority
```

Comment:
Tiny formatting idea: It looks like `Url` appears once before in the document as `URL`, so it could be capitalized here as `URL` too. Then it would also match the capitalization from <https://docs.oracle.com/javase/8/docs/technotes/guides/security/time-of-signing.html>